docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -148,10 +148,14 @@ This means you can define multiple DAGs per Python file, or even spread one very
 
 Note, though, that when Airflow comes to load DAGs from a Python file, it will only pull any objects at the *top level* that are a DAG instance. For example, take this DAG file::
 
-    dag_1 = DAG('this_dag_will_be_discovered')
+    with DAG('this_dag_will_be_discovered') as dag:
```

Comment:
This whole section is wrong with `auto_register` now :(

Can you refactor this whole section?

Reply:
@fritz-astronomer Any chance these `as dag:` bits could be removed as @jedcunningham mentioned?